study-builder/studies/pancan/snippets/question-primary-cancer-self.conf

```diff
@@ -0,0 +1,10 @@
+{
+  include required("../../snippets/picklist-question-single-autocomplete.conf"),
+  include required("common/cancer-picklist-options.conf"),
```

Comment:
I'm glad we pulled out the list of options themselves so we can share it, otherwise this PR will be huge!